lib/validations.js

```diff
@@ -628,6 +628,7 @@ function blank(v) {
   if (typeof v === 'undefined') return true;
   if (v instanceof Array && v.length === 0) return true;
   if (v === null) return true;
+  if (typeof v === 'number' && isNaN(v)) return true;
```

Comment:
Note that `isNaN("NaN")` returns true, that's why we need to check the type. `Number.isNaN` was introduced in ES6 AFAIK, thus it is not available in ES5 browsers (like phantomjs 1.x).